Xwt.XamMac/Xwt.Mac/DialogBackend.cs

```diff
@@ -169,6 +169,13 @@ public void RunLoop (IWindowFrameBackend parent)
 		{
 			Visible = true;
 			modalSessionRunning = true;
+
+			if (parent.Window is NSWindow nsParent)
```

Comment:
Shoudl we add it as a child window if the parent is not visible?

Reply:
fixed